_returners/splunk_nebula_return.py

```diff
@@ -308,17 +304,17 @@ def flushBatch(self):
 
         if len(self.batchEvents) > 0:
             headers = {'Authorization': 'Splunk ' + self.token}
-            self.server_uri = [x for x in self.server_uri if x[1] is not "bad"]
+            self.server_uri = [x for x in self.server_uri if x[1] is not False]
             for server in self.server_uri:
                 try:
                     r = requests.post(server[0], data=' '.join(self.batchEvents), headers=headers, verify=http_event_collector_SSL_verify, proxies=self.proxy, timeout=self.timeout)
                     r.raise_for_status()
-                    server[1] = "good"
+                    server[1] = True
                     break
                 except requests.exceptions.RequestException:
-                    log.error('Request to splunk server "%s" failed. Marking as bad.' % server[0])
-                    server[1] = "bad"
+                    log.info('Request to splunk server "%s" failed. Marking as bad.' % server[0])
+                    server[1] = False
                 except Exception as e:
-                    log.error('Request to splunk threw an error: {0}'.format(e))
+                    log.info('Request to splunk threw an error: {0}'.format(e))
```

Comment:
The biggest problem right now is that returner errors do not get forwarded to the master, just the job results, so you won't see anything. There's also not a really good mechanism for returning those errors....we have the event bus, to which we could fire events, but then we'd have to have a reactor in place to actually consume those events, or they just get lost....

We also have more and more people using the "saltless" hubble, and any master-related solution to the error reporting won't help them. I don't have a good solution yet.